doc/DeveloperGuide.md

```diff
@@ -59,6 +59,8 @@ Priority | As a ... | I want to ... | So that I can...
 `* * *` | user | add a new person |
 `* * *` | user | delete a person | remove entries that I no longer need
 `* * *` | user | find a person by name | locate details of persons without having to go through the entire list
+`* * *` | user | edit a person's details by index | add and remove entries every time a person's details change.
```

Comment:
perhaps  what you meant for `So that I can add and remove entries every time a person's details change` is actually `So that I can easily change a person details without having to add and remove entries every time`?

Note it your user stories should follow the `| As a ... | I want to ... | So that I can...` template as given.